packages/hooks/LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License
```

Comment:
this file is unneeded now that https://github.com/Developer-DAO/web3-ui/pull/29 is merged